index.html

```diff
@@ -22,58 +22,81 @@
   <body>
     <section class="container">
       <header class="upper">
-        <nav>
+        <nav class="navigator">
+          <a href="">Hello</a>
+          <a href="">portofolio</a>
+          <a href="">About</a>
+          <a href="">Contact</a>
           <img
             class="image"
             src="assests/images/ic_menu.png"
             alt="the menu icon"
           />
         </nav>
       </header>
-      <section class="content">
-        <h1 class="my-name">Bassem Shams</h1>
-        <p class="paragraph">
-          Hello! I am a software developer! I can help you build a product,
-          feature or website. Take a look of my works. If you like what you see
-          and have a project you need coded, do not hesitate and contact me.
-        </p>
-        <button class="button">Start collaboration</button>
+      <section class="hello-content">
+        <article class="content">
+          <h1 class="my-name">Bassem Shams</h1>
+          <p class="paragraph">
+            Hello! I am a software developer! I can help you build a product,
+            feature or website. Take a look of my works. If you like what you
+            see and have a project you need coded, do not hesitate and contact
+            me.
+          </p>
+          <button class="button">Start collaboration</button>
+        </article>
+        <footer class="lower"></footer>
       </section>
-      <section class="lower"></section>
       <!--the second part of the project-->
-      <section class="container-projects">
-        <h2 class="header-project">projects</h2>
-        <section class="project">
-          <div>
+      <section class="project">
+        <div class="back1"></div>
+        <div class="back2"></div>
+        <div class="back3"></div>
+        <div class="back4"></div>
+
+        <article class="projects">
+          <h2 class="header-project">projects</h2>
```

Comment:
- [ ] Good job on the styles but for this project you are supposed to stick to the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9)template as much as possible. Please use the correct font-weight for your `projects` and `about` headings and make sure they match the design on the [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9).

### Your design
![image](https://user-images.githubusercontent.com/83803414/160180328-731a1d53-9ead-49cc-bd33-a6dffeb84b4d.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/160180386-f82bbadb-deaa-4bcb-a21b-618681fdc499.png)